client/package.json

```diff
@@ -13,6 +13,7 @@
     "babel-loader": "^9.2.1",
     "bootstrap-icons": "^1.11.3",
     "open-location-code": "^1.0.3",
+    "prettier": "^3.3.3",
```

Comment:
Why do we need this?

Reply:
Cross IDE prettier config stays same.
But I will move it to devDepencies